projects/client/src/lib/requests/models/SocialActivity.ts

```diff
@@ -17,12 +17,14 @@ export const SocialActivityEpisodeSchema = z.object({
   activityAt: z.date(),
   type: z.literal('episode'),
   users: z.array(UserProfileSchema),
-  show: ShowEntrySchema,
-  episode: EpisodeEntrySchema,
+  episode: EpisodeEntrySchema.merge(z.object({
+    show: ShowEntrySchema,
+  })),
```

Comment:
I don't know why this is better (aka: revert this pointless merge please, it's harder to read).